xbmc/filesystem/VideoDatabaseDirectory/DirectoryNodeSeasons.cpp

```diff
@@ -31,21 +31,33 @@ std::string CDirectoryNodeSeasons::GetLocalizedName() const
 {
   switch (GetID())
   {
-  case 0:
-    return g_localizeStrings.Get(20381); // Specials
-  case -1:
-    return g_localizeStrings.Get(20366); // All Seasons
-  case -2:
-  {
-    CDirectoryNode *pParent = GetParent();
-    if (pParent)
-      return pParent->GetLocalizedName();
-    return "";
-  }
-  default:
-    std::string season =
-        StringUtils::Format(g_localizeStrings.Get(20358), GetID()); // Season <season>
-    return season;
+    case 0:
+      return g_localizeStrings.Get(20381); // Specials
+    case -1:
+      return g_localizeStrings.Get(20366); // All Seasons
+    case -2:
+    {
+      CDirectoryNode* pParent = GetParent();
+      if (pParent)
+        return pParent->GetLocalizedName();
+      return "";
+    }
+    default:
+    {
+      std::string season;
```

Comment:
maybe wrap this into a new private method to avoid spreading db code on common logic

Reply:
changed not sure that is what you mean